frontend/__tests__/containers/MetricsStepper/MetricsStepper.test.tsx

```diff
@@ -298,10 +298,12 @@ describe('MetricsStepper', () => {
     const expectedJson = {
       board: undefined,
       calendarType: 'Regular Calendar(Weekend Considered)',
-      dateRange: {
-        endDate: null,
-        startDate: null,
-      },
+      dateRange: [
```

Comment:
Not compatible right now, system will throw the alert `Imported data is not perfectly matched. Please review carefully before going next!`.

We can discuss if its necessary to change this part of logic to accept `dateRange: { startDate, endDate }` for a period of time.

Reply:
Align with BA firstly，tech solution is ok for dev，it will impact the next release version number

Reply:
so you change the UT and e2e test case, need to keep origin and add your test case

Reply:
Of course these temporary `{ startDate, endDate }` to `{ startDate, endDate }[]` is just make it possible to pass the existing e2e tests(it also make sense for existing cases, since range[0] means legacy timerange).

Apparently we'd better to design and add comprehensive time range e2e flow when the whole flow is developing done. It's not timing now.